docs/guides/quickstart.rst

```diff
@@ -0,0 +1,61 @@
+Quickstart
+==========
+
+Installation
+~~~~~~~~~~~~
+
+This guide teaches how to use Py-EVM as a library. For contributors, please check out the
+:doc:`Contributing Guide </contributing>` which explains how to set everything up for development.
+
+
+Installing on Ubuntu
+--------------------
+
+Py-EVM requires Python 3.6 as well as some tools to compile its dependencies. On Ubuntu, the
```

Comment:
@pipermerriam I copied this from Trinity and left Python 3.6 on purpose as we plan to drop Python 3.5 support anyway (so why not start preparing people ;))